tests/spec/core/data-tests-spec.js

```diff
@@ -17,6 +17,11 @@ describe("Core — data-tests attribute", () => {
             <a id="t1" data-cite="#test">a</a>
             <dfn id="t2" data-cite="#test">a</dfn>
           </p>
+          <p id="metadata-tests"
+            data-tests="this-is-secure.https.html,
+             this-is-manual.html, this-is-secure-and-manual.https.html">
```

Comment:
Nit: put "this-is-secure-and-manual.https.html" on a new line, so it's visually distinct from the other two. Basically, we want:
```HTML
data-tests="
   this-is-secure.https.html,
   this-is-manual.html,
   this-is-secure-and-manual.https.html"
```